model/functions.php

```diff
@@ -660,4 +660,280 @@ function getAllColumbariumRelatedDataWithFilter(ColumbariumFilter $filter)
+        }
+        else{
+            $statement->bindValue(':obituary', $obituary = $buriedIndividual->obituary);
+        }
+        if(!isset($buriedIndividual ->tombId))
+        {
+            $statement->bindValue(':tombId', null, PDO::PARAM_NULL);
+        }
+        else{
+            $statement->bindValue(':tombid', $tombId = $buriedIndividual->tombId);
+        }
+        if(!isset($buriedIndividual ->columbariumId))
+        {
+            $statement->bindValue(':columbariumId', null, PDO::PARAM_NULL);
+        }
+        else{
+            $statement->bindValue(':columbariumId', $columbariumId = $buriedIndividual->columbariumId);
+        }
+        $success = $statement->execute();
+        $statement->closeCursor();
+        if($success)
+        {
+            $response->addResult(True);
+        }
+        else{
+            $response->addError("Failed to insert Buried Individual.");
+        }
+    } catch (PDOException $e) {
+        $errorMessage = $e->getMessage();
+        $response -> addError($errorMessage);
+    }
+    return $response;
+}
+
+
+function insertColumbariumType(ToTypeTable $type)
+{
+    $response = new Response();
+    try{
+        $db = connection::getInstance();
+        $con = $db -> get_connection();
+        $query = "INSERT INTO columbarium_types (TYPE)
+                    VALUE (:columbariumType);";
+        $statement = $con->prepare($query);
+        $statement->bindValue(':columbariumType', $type->type);
+        $success = $statement->execute();
+        $statement->closeCursor();
+        if($success)
+        {            
+            $response->addResult(True);
+        }
+        else{
+            $response->addError("Failed to insert Columbarium Type.");
+        }
+    } catch (PDOException $e) {
+        $errorMessage = $e->getMessage();
+        $response->addError($errorMessage);
+    }
+    return $response;
+}
+
+function insertOwner(ToOwnerTable $owner)
+{
+    $response = new Response();
+    try{
+        $db = connection::getInstance();
+        $con = $db -> get_connection();
+        $query = "INSERT INTO owners (FIRST_NAME, LAST_NAME, MIDDLE_NAME, ADDRESS, PHONE_NUMBER, EMAIL)
+                    VALUES(:firstName, :lastName, :middleName, :address, :phoneNumber, :email);";
+        $statement = $con->prepare($query);
+        $statement->bindValue(':firstName', $owner->firstName);
+        $statement->bindValue(':lastName', $owner->lastName);
+        if(!isset($owner->middleName))
+        {
+            $statement->bindValue(':middleName', null, PDO::PARAM_NULL);
+        }
+        else
+        {
+            $statement->bindValue(':middleName', $middleName = $owner->middleName);
+        }
+        $statement->bindValue(':address', $address = $owner->address);
+        $statement->bindValue(':phoneNumber', $phoneNumber = $owner->phoneNumber);
+        $statement->bindValue(':email', $email = $owner->email);
```

Comment:
Lines 808 - 810
address, phone number, and email are all nullable. Make appropriate changes and remove the assignments in the second parameters for each one of them.